WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -472,40 +464,19 @@ - (ReaderCoordinator *)readerCoordinator
     }
 
     
-    if ([Feature enabled:FeatureFlagFloatingCreateButton]) {
-        [allViewControllers removeObject:self.newPostViewController];
-    }
+    [allViewControllers removeObject:self.newPostViewController];
```

Comment:
maybe we can just remove `newPostViewController` and `meSplitViewController` entirely?